### PR TITLE
Add ellipsis only if the source string is longer than the limit

### PR DESCRIPTION
**Pull request by gabriel-glo**

### CODE REVIEW

File: theme/utils/class-excerpt.php [hunk 1]
```diff
@@ -50,8 +47,13 @@ public static function get_excerpt( $source = null, $limit = null ) {
       $output = substr( $output, 0, strripos( $output, ' ' ) );
     }
 
-    $output = '<p>' . $output . '...</p>';
+    // Reduce string to limit.
+    $output = wp_trim_words( $output, $limit, '...' );
+
+    $output = '<p>' . $output . '</p>';
```
Comment (dingo-d) [inline theme/utils/class-excerpt.php]: Should we force a paragraph here? What if someone wants to use `h3` for their excerpts? Maybe we should remove this and let the actual output in the templates worry about this?

Thoughts @infinum/eightshift-team ?

Comment (iruzevic) [reply theme/utils/class-excerpt.php]: agreed 👍

Comment (gabriel-glo) [reply theme/utils/class-excerpt.php]: WordPress wraps its output in paragraph tags. Both `the_excerpt()` and `get_the_excerpt()` functions do this.

Comment (gabriel-glo) [reply theme/utils/class-excerpt.php]: Personally, I'm for removing paragraph tags. More than once I've had issues with styling excerpt and needing to reference the actual paragraph to style it properly.

Comment (dingo-d) [reply theme/utils/class-excerpt.php]: Yeah, but we are changing the workings of the excerpt function so we can control it. I'm for removal 🙂

**Comment by dingo-d** [comment]
Please change the base branch to `development` instead of `master` before merging this.

**Pull request merged**